shell/platform/embedder/BUILD.gn

```diff
@@ -87,6 +87,12 @@ embedder_source_set("embedder_with_symbol_prefix") {
   public_configs = [ ":embedder_prefix_config" ]
 }
 
+source_set("embedder_headers") {
```

Comment:
You should be able to make the templated `source_set(target_name) {` on line 24 have `public_deps = [ ":embedder_headers" ]` and then remove embedder.h from its sources list, so this header isn't part of two different targets.